bot/seasons/evergreen/fun.py

```diff
@@ -0,0 +1,40 @@
+import logging
+import random
+
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+
+class Fun:
+    """
+    A collection of general commands for fun.
+    """
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command()
+    async def roll(self, ctx, str_input: str = None):
+        output = ""
+
+        if not str_input:
+            output = "To use .roll, format it as such: .roll (number)"
+        else:
+            try:
+                num_rolls = int(str_input)
+                if num_rolls > 6:
+                    num_rolls = 6
+                elif num_rolls < 1:
+                    return
+                for i in range(num_rolls):
```

Comment:
Due to `i` not being used, it should either be named `_` or `_i` to signify it's an intentionally unused variable. Currently the linter is failing on this.

Reply:
(we tend to prefer `_`)

```suggestion
                for _ in range(num_rolls):
```